site/app/templates/grading/electronic/PDFAnnotationEmbedded.twig

```diff
@@ -67,12 +67,17 @@
     same could not be achived through CSS as it cause the tools to hide when moved right in panel.
     This has been done taking care of resizing of panel ,platform(device),overlapping and scrolling of panels */
 
-    const offSetter = new ResizeObserver(entries => {
-        entries.forEach(entry => {
-            let height = $(".sticky-file-info").outerHeight();
-            $("#viewer").css("paddingTop", height);
+    try {
+        const offSetter = new ResizeObserver(entries => {
+            entries.forEach(entry => {
+                let height = $(".sticky-file-info").outerHeight();
+                $("#viewer").css("paddingTop", height);
+            });
         });
-    });
-    offSetter.observe(document.querySelector(".sticky-file-info"));
+        offSetter.observe(document.querySelector(".sticky-file-info"));
+    }
+    catch (e) {
+        // Prevent defining the variable again
+    }
```

Comment:
Is there a reason why we need these global variables in the first place?  Ideally this could be better fixed by eliminating the global variables.  If the global variables are absolutely necessary, it would be better to use [`typeof` to check if a variable is defined](https://stackoverflow.com/questions/5113374/javascript-check-if-variable-exists-is-defined-initialized) instead of a try-catch.  As it is now, if another exception is thrown in that block of code, nothing will ever be printed in the console, which will make future development challenging.